sdks/java/core/src/main/java/org/apache/beam/sdk/util/RowJson.java

```diff
@@ -86,7 +88,7 @@
 })
 public class RowJson {
   private static final ImmutableSet<TypeName> SUPPORTED_TYPES =
-      ImmutableSet.of(BYTE, INT16, INT32, INT64, FLOAT, DOUBLE, BOOLEAN, STRING, DECIMAL);
+      ImmutableSet.of(BYTE, INT16, INT32, INT64, FLOAT, DOUBLE, BOOLEAN, STRING, DECIMAL, DATETIME);
```

Comment:
Do we need to use the DATETIME primitive type for support in Calcite SQL? I thought ZetaSQL used the logical MicrosInstant type.

Reply:
ZetaSQL haven't fully migrated to MicroInstant yet, so this primitive DATETIME type is used for both CalciteSQL and ZetaSQL now.